src/features/events/components/newsletter/newsletter.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import NewsletterForm from './'
+
+describe('<NewsletterForm/>', () => {
+  beforeAll(() => {
+    global.___loader = {
+      enqueue: jest.fn(),
+    }
+    global.fetch = jest.fn().mockImplementation(() => Promise.resolve({}))
+  })
+  afterAll(() => {
+    global.___loader.enqueue.mockReset()
+    global.fetch.mockClear()
+    delete global.fetch
+    wrapper.unmount()
```

Comment:
This `wrapper` is being unmounted before declaration. In general within this file, the scoping needs to be a little tighter with the usages of multiple instances of `wrapper`. These should be scoped off in separate `it` or `describe` blocks and only mounted when a mount is required. For example, in this file we'd be aiming for something like this:

```
it('should match snapshot`, () => {
  const wrapper = ...
})

describe('wrapper manipulation', () => {
  const wrapper = ...

  it('should use the wrapper', () => {
    expect(wrapper.blah).toExist()
  })
})
```

In addition, following this pattern we won't need to call an `unmount` on the wrapper as the GC will collect it on every block exit. This actually happens within the global scope of Jest anyway, so it's not something we need to worry about too much.

Reply:
Noted for next time! Deleted tests per our conversation about this component being deprecated for time being.